apps/site/src/lib/components/Footer/Footer.tsx

```diff
@@ -50,15 +51,17 @@ const Footer = () => {
 		<div className="flex flex-row gap-1 items-center mt-2 max-[600px]:flex-col max-[210px]:gap-0 max-[210px]:mt-0">
 			<div className="flex flex-row gap-1 items-center max-[210px]:flex-col max-[210px]:gap-0">
 				<p className="m-0">Made with</p>
-				<svg className=" fill-white" xmlns="http://www.w3.org/2000/svg" height="16" width="16" viewBox="0 0 512 512"><path d="M47.6 300.4L228.3 469.1c7.5 7 17.4 10.9 27.7 10.9s20.2-3.9 27.7-10.9L464.4 300.4c30.4-28.3 47.6-68 47.6-109.5v-5.8c0-69.9-50.5-129.5-119.4-141C347 36.5 300.6 51.4 268 84L256 96 244 84c-32.6-32.6-79-47.5-124.6-39.9C50.5 55.6 0 115.2 0 185.1v5.8c0 41.5 17.2 81.2 47.6 109.5z"/></svg>
+				<Image src={redHeart} alt="Red heart icon" layout="intrinsic" />
+
 				<p className="m-0">in Irvine, CA <span className="max-[600px]:hidden">•</span></p>
```

Comment:
Due to Student Center regulations, we have to specify that IrvineHacks is hosted by us, Hack at UCI, so switch the text of the footer to say "Made with ❤️ by Hack at UCI ..."

And that means we also have to add our Hack logo linked to our club site.